todo/input.py

```diff
@@ -209,17 +209,19 @@ def smart_menu(card, f_display, list_choices, label_choices, color=None):
     @staticmethod
     def title_to_link(card):
         # assumes card.name is the link you want
-        links = [n for n in card.name.split() if 'http' in n]
+        sp = card.name.split()
+        links = [n for n in sp if 'http' in n]
         existing_attachments = [a.name for a in card.get_attachments()]
         for l in links:
             if l not in existing_attachments:
                 card.attach(url=l)
-        # attempt to get the title of the link
+        # attempt to get the title of the link for a HTML page
         possible_title = get_title_of_webpage(links[0])
```

Comment:
When adding in the formatting, it would be cool to get access to the titles of all links via an indexable parameter

Reply:
Agreed! I implemented a little template system that lets you reference the title of each link individually using variables like `$link0` or `$title0`. I'll attach a screencap of the interface in use:
![for_jason](https://user-images.githubusercontent.com/3670674/37238113-502de704-23ed-11e8-8500-765865d24aa8.png)